src/meson.build

```diff
@@ -56,6 +56,8 @@ elif cpu_family == 'x86_64' and endian == 'little'
   dynamic_linker = '/lib64/ld-linux-x86-64.so.2'
 elif cpu_family == 'riscv64' and endian == 'little'
   dynamic_linker = '/lib/ld-linux-riscv64-lp64d.so.1'
+elif cpu_family == 'loongarch64' and endian == 'little'
+  dynamic_linker = '/lib/ld-linux-loongarch-lp64d.so.1'
```

Comment:
Is there a typo here?

According to [this](https://sourceware.org/glibc/wiki/ABIList#loongarch) ABI specification, the dynamic linker should be `/lib64/ld-linux-loongarch-lp64d.so.1`, but here it's only `/lib/...`.

Reply:
It was a clerical error, you're right, thanks for the code review!

Reply:
Okay!  Thanks for confirming that.